<?xml version="1.0" encoding="UTF-8"?><testrun name="BookstoreRepositoryTest" project="bookstore" tests="9" started="9" failures="1" errors="0" ignored="0">
  <testsuite name="fi.hh.palvelinohjelmointi.bookstore.bookstore.BookstoreRepositoryTest" time="1.74">
    <testcase name="deleteCategory" classname="fi.hh.palvelinohjelmointi.bookstore.bookstore.BookstoreRepositoryTest" time="1.314"/>
    <testcase name="findByNameShouldReturnBook" classname="fi.hh.palvelinohjelmointi.bookstore.bookstore.BookstoreRepositoryTest" time="0.064"/>
    <testcase name="findByNameShouldReturnUser" classname="fi.hh.palvelinohjelmointi.bookstore.bookstore.BookstoreRepositoryTest" time="0.044"/>
    <testcase name="findByNameShouldReturnCategory" classname="fi.hh.palvelinohjelmointi.bookstore.bookstore.BookstoreRepositoryTest" time="0.026"/>
    <testcase name="createCategory" classname="fi.hh.palvelinohjelmointi.bookstore.bookstore.BookstoreRepositoryTest" time="0.063"/>
    <testcase name="createBook" classname="fi.hh.palvelinohjelmointi.bookstore.bookstore.BookstoreRepositoryTest" time="0.041"/>
    <testcase name="createUser" classname="fi.hh.palvelinohjelmointi.bookstore.bookstore.BookstoreRepositoryTest" time="0.122">
      <failure>java.lang.AssertionError: &#13;
Expecting:&#13;
 &lt;fi.hh.palvelinohjelmointi.bookstore.bookstore.domain.User@250d440&gt;&#13;
to be in:&#13;
 &lt;["user2"]&gt;&#13;
&#13;
	at fi.hh.palvelinohjelmointi.bookstore.bookstore.BookstoreRepositoryTest.createUser(BookstoreRepositoryTest.java:88)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)&#13;
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)&#13;
	at org.springframework.test.context.junit4.statements.RunBeforeTestMethodCallbacks.evaluate(RunBeforeTestMethodCallbacks.java:75)&#13;
	at org.springframework.test.context.junit4.statements.RunAfterTestMethodCallbacks.evaluate(RunAfterTestMethodCallbacks.java:86)&#13;
	at org.springframework.test.context.junit4.statements.SpringRepeat.evaluate(SpringRepeat.java:84)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)&#13;
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:252)&#13;
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:94)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)&#13;
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:70)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:191)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="deleteUser" classname="fi.hh.palvelinohjelmointi.bookstore.bookstore.BookstoreRepositoryTest" time="0.028"/>
    <testcase name="deletebook" classname="fi.hh.palvelinohjelmointi.bookstore.bookstore.BookstoreRepositoryTest" time="0.038"/>
  </testsuite>
</testrun>
